e2ev2/tests/sanity.browser.test.js

```diff
@@ -13,19 +13,24 @@ browserTest('page loads', async browser => {
 browserTest('view cookie policy page', async browser => {
   const page = browser.initialPage
   await tu.useApiProxy(page)
-  await page.goto(config.urlRoot()+'/login', {waitUntil: 'networkidle0'})
-  const cpLink = await page.waitForSelector('a[href="/cookie-policy"]')
-  expect(cpLink).toBeDefined()
-  // click and wait need to happen simultaneously to avoid a race
-  const [, target] = await Promise.all([
-    cpLink.click(),
-    browser.waitForTarget(target => target.opener() === page.target(), {timeout: 2e3})
-  ])
-  const cpPage = await target.page()
-  cpPage.setDefaultTimeout(2000)
-  const h3 = await cpPage.waitForSelector('h3')
-  expect(await h3.evaluate(n => n.innerText))
-    .toBe('All of Us Research Program Cookie Policy')
+    .then(async () => await page.goto(config.urlRoot()+'/login', {waitUntil: 'networkidle0'}))
+    .then(async () => {
```

Comment:
What is the logic for splitting up the `await` calls in 2 `then`s like this?  Naively I would either expect a single `then` or many `then`s

Reply:
I believe that you are right, one then seems like the more intuitive approach. I updated it.